src/ripple/proto/ripple.proto

```diff
@@ -210,9 +214,14 @@ message TMValidatorList
 // Used to sign a final closed ledger after reprocessing
 message TMValidation
 {
-    required bytes validation       = 1;    // in STValidation signed form
-    optional bool checkedSignature  = 2;    // node vouches signature is correct
-    optional uint32 hops            = 3;    // Number of hops traveled
+    // The serialized validation
+    required bytes validation = 1;
+
+    // node vouches signature is correct
+    optional bool checkedSignature = 2 [deprecated = true];
+
+    // Number of hops traveled
+    optional uint32 hops = 3           [deprecated = true];
```

Comment:
Could `hops` still be useful for the `reduce relaying` task? We can infer the preferred peer to receive TMValidation and TMPropose from as the ones with the smallest number of hops.